tensorflow/lite/micro/kernels/exp.cc

```diff
@@ -12,78 +12,69 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+
+#include "tensorflow/lite/kernels/internal/reference/exp.h"
+
 #include "tensorflow/lite/c/common.h"
-#include "tensorflow/lite/kernels/internal/reference/reference_ops.h"
-#include "tensorflow/lite/kernels/internal/tensor.h"
 #include "tensorflow/lite/kernels/internal/tensor_ctypes.h"
 #include "tensorflow/lite/kernels/kernel_util.h"
+#include "tensorflow/lite/micro/kernels/kernel_util.h"
 
 namespace tflite {
 namespace ops {
-namespace builtin {
+namespace micro {
 namespace exp {
```

Comment:
we're going with a flat namespace in micro.

See https://github.com/tensorflow/tensorflow/commit/b13e49836a92241d653e96225552bd8df3bfd92e and https://github.com/tensorflow/tensorflow/blob/b0fcce8080f94b44195c429929361a420e9255cc/tensorflow/lite/micro/kernels/conv.cc

Tagging @rkuester to make sure that this is part of the updated OP porting guide.

Reply:
Indeed [it is](https://github.com/rkuester/tensorflow/blame/e6010bf77a4b9341f1ddd7fe85d6e07a3d75e843/tensorflow/lite/micro/kernels/PORTING.md#L60).

Reply:
changed to flat namespace.